client/src/pages/Student.js

```diff
@@ -141,6 +155,7 @@ class Student extends Component {
                       value={this.state.student.hoursAvail}
                       onChange={this.changeHandler}
                   />
+                  {this.validator.message('hoursAvail', this.state.student.hoursAvail, 'required|min:10|max:20')}
```

Comment:
This validator is actually asking user to enter a string with length between 10 and 20 characters, instead of a number in between 10-20